sdk-autoconfigure-support/src/test/java/io/opentelemetry/instrumentation/resources/ResourceProviderPropertiesCustomizerTest.java

```diff
@@ -119,6 +119,10 @@ Stream<DynamicTest> enabledTestCases() {
                             Boolean.toString(tc.explicitEnabled));
                       }
 
+                      props.put("otel.traces.exporter", "none");
+                      props.put("otel.metrics.exporter", "none");
+                      props.put("otel.logs.exporter", "none");
+
                       Attributes attributes =
                           SdkAutoconfigureAccess.getResourceAttributes(
```

Comment:
Perhaps it would be better to copy `SdkAutoconfigureAccess` under test sources. Moving that class to this module makes it part of the public api.